README.md

```diff
@@ -7,11 +7,12 @@ A demo of Python's project with automated tests.
 How to install?
 ===============
 1. Create separate virtual environment with Python 3.5.x.
-2. Run `pip install -r requirements.txt` to install required Python's dependencies.
+2. Run `pip install -r requirements-dev.txt` to install required Python's dependencies.
 
-Howe to run automated tests?
+How to run automated tests?
 ============================
-Run `py.test demo_python_at` within your virtual environment.
+1. Run `py.test demo_python_at` within your virtual environment.
+2. Run `py.test --cov=demo_python_at tests/` within your virtual environment to run tests with coverage. 
```

Comment:
This command runs unit tests, but this section about automated tests in `demo_python_at`. Please restore this paragraph.